llvm/lib/Target/AArch64/AArch64Subtarget.cpp

```diff
@@ -144,7 +144,6 @@ void AArch64Subtarget::initializeProperties(bool HasMinSize) {
   case CortexA78C:
   case CortexR82:
   case CortexX1:
-  case CortexX1C:
```

Comment:
Seems like a stray change, if not, why remove just this one?

Reply:
Ah, does the CortexX1C have its properties described in tablegen already, as the comment at the top of the function suggests we do?

Reply:
For AArch64 specifically, there is no `SubtargetFeature` corresponding to `CortexX1C`, and therefore `ARMProcFamily` can never be set to this value. There were a few enum values like this that could never be set.